app/controllers/hiring_staff/organisations/user_preference_controller.rb

```diff
@@ -0,0 +1,39 @@
+class HiringStaff::Organisations::UserPreferenceController < HiringStaff::BaseController
+  before_action :verify_school_group
+
+  def show
+    @user_preference_form = UserPreferenceForm.new(current_user, current_organisation)
+  end
+
+  def update
+    @user_preference_form = UserPreferenceForm.new(current_user, current_organisation, user_preferences_form_params)
+
+    if @user_preference_form.valid?
+      @user_preference_form.save
+      return redirect_to organisation_path
+    end
+
+    render :show
```

Comment:
If you prefer it this way I will not add this comment anymore, I still think that an if/else statement with redirect on success and render on error is more clear to read and more in line with the standard rails way, like when you use a generator to scaffold a resource

Reply:
It's just habit (same as always returning from methods), I'm happy to roll the other way